cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -2111,7 +2114,7 @@ makeShelleyTransactionBody era@ShelleyBasedEraAllegra
         scripts
         TxBodyNoScriptData
         txAuxData
-        txScriptValidity
+        (unBuildTxWith txScriptValidity)
```

Comment:
Same here and for all eras except Alonzo.

Reply:
Done!